### ISSUE TITLE
top/left pixels are missing when the input width/height is odd-number

### ISSUE THREAD

**Issue by nagadomi** [alexjc/neural-enhance#76]
With `--zoom=2` option,

input: 160x240, output: 320x480 # OK
input: 321x481, output: 640x960 # I expected 642x962

**Comment by alexjc** [comment]
Depending on the internal network architecture (if it downscales), you need input images multiples of 2 or 4 pixels for the output to be the same size.

**Comment by nagadomi** [comment]
This issue occurs with `python enhance.py --device=gpu0 --zoom 2 --type photo --model default input.png`.
I am not familiar with this code base, but I think this issue is able to fix it using padding input/cropping output.

**Comment by alexjc** [comment]
You're right, with the new tiled rendering since 'v0.2' any number of pixels should be possible.
